src/hotspot/share/classfile/classListParser.cpp

```diff
@@ -461,7 +461,6 @@ bool ClassListParser::is_matching_cp_entry(constantPoolHandle &pool, int cp_inde
 }
 
 void ClassListParser::resolve_indy(Symbol* class_name_symbol, TRAPS) {
-
```

Comment:
Line deleted by accident?

Reply:
It's uncommon to have a blank line at the beginning of a function, so I deleted it.